.gitignore

```diff
@@ -31,3 +31,11 @@ cache
 tsconfig.tsbuildinfo
 
 dist/
+
+# Ignore the blog-data json that we generate during dev and build
+apps/site/public/blog-data.json
+
+# Ignore worker artifacts
+apps/site/.open-next
+apps/site/.wrangler
+
```

Comment:
Should these be added to our other ignore files, like prettier, etc?

Reply:
I'm not really sure to be honest

I know that prettier does ignore the files specified in the `.gitignore` by itself, and `npm run lint` doesn't seem to pick up the files inside `.open-next` and `.wrangler` (I am not completely sure what is making it ignore those) so I'm not sure if any extra settings are actually necessary 🤔

Reply:
Still, I think it'd be good to include it in `.stylelintignore`.